bold_element.py

```diff
@@ -0,0 +1,12 @@
+from element import Element
+import re
+
+class BoldElement(Element):
+
+    def _render(self):
+        pattern = re.compile(r'[*]{2}(.*?)[*]{2}')
+        rendered = pattern.sub(r'<strong>\1</strong>', self.content())
```

Comment:
[Opt]: better have a space between label and content.

Reply:
You mean `<strong> \1 </strong>` rather than `<strong>\1</strong>`, right?

Reply:
Yes, but its totally optional.

Reply:
> I think the whitespace matters. `my<strong>test</strong>` and `my<strong> test </strong>` are two different things.

that's right. there would be extra space in the html text